week-2/stretch/count-words.js

```diff
@@ -26,3 +26,31 @@
 
 3. Order the results to find out which word is the most common in the chapter
 */
+function countWords(words) {
+  const count = {};
+  const wordSplit = words.split(" ");
+  const lower = wordSplit.map(word => word.toLowerCase());
+
+  for (let i = 0; i < wordSplit.length; i++) {
```

Comment:
This would be a great place to use a [`for...of` loop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) 🙂